### PR TITLE
Bugfix FXIOS-6805 [v116] Update padding and corners on 'Use saved card' button

### PR DESCRIPTION
**Pull request by PARAIPAN9**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-6805)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/15155)

### Description
Update padding and add corner radius to the Use saved card accessory view

screenshot

### Pull requests checks where applicable
- [x] Fill in the three TODOs above (tickets number and description of your work)
- [x] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [ ] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by PARAIPAN9** [comment]
@jeffreygee for UX

File: Client/AccessoryViewProvider.swift [hunk 1]
```diff
@@ -143,8 +139,18 @@ class AccessoryViewProvider: UIView, Themeable {
...
+            spacer.widthAnchor.constraint(equalToConstant: width),
+            spacer.heightAnchor.constraint(equalToConstant: 30)
+        ])
+        spacer.accessibilityElementsHidden = true
+    }
+
     private func setupLayout() {
         translatesAutoresizingMaskIntoConstraints = false
+        setupSpacer(leadingFixedSpacer, width: 2)
+        setupSpacer(trailingFixedSpacer, width: 3)
```
Comment (nbhasin2) [inline Client/AccessoryViewProvider.swift:153]: ⚠️ nit: Would be good to add 2 and 3 as part of the UX struct

File: Client/AccessoryViewProvider.swift [hunk 2]
```diff
@@ -143,8 +139,18 @@ class AccessoryViewProvider: UIView, Themeable {
         layoutIfNeeded()
     }
 
+    private func setupSpacer(_ spacer: UIView, width: CGFloat = 0) {
+        NSLayoutConstraint.activate([
+            spacer.widthAnchor.constraint(equalToConstant: width),
+            spacer.heightAnchor.constraint(equalToConstant: 30)
```
Comment (nbhasin2) [inline Client/AccessoryViewProvider.swift:145]: ⚠️ nit: Would be good to add 30 as part of the UX struct

**Comment by jeffreygee** [comment]
> 

Updated padding and corners look good. Thanks!

**Review by nbhasin2 — APPROVED** [review]

**Pull request merged**